README.md

```diff
@@ -10,7 +10,7 @@
 
 * [Get started](#get-started)
 
-* [Installing Akamai Pipeline](#installing-promotional-deployment)
```

Comment:
"#installing-promotional-deployment" needs to be changed to "#installing-akamai-pipeline"